net-proxy/trojan/trojan-1.16.0-r2.ebuild

```diff
@@ -50,6 +50,9 @@ src_prepare() {
 src_configure() {
 	local mycmakeargs=(
 		-DENABLE_MYSQL=$(usex mysql)
+		-DENABLE_NAT=$(usex nat)
```

Comment:
These add no new dependencies. Why not enable them unconditionally? Are they configurable at runtime?